Podfile

```diff
@@ -6,6 +6,8 @@ target 'SwedbankPaySDK' do
   use_frameworks!
 
   # Pods for SwedbankPaySDK
+  pod 'Alamofire', '~> 4.9.0'
+  pod 'AlamofireObjectMapper', '~> 5.0'
```

Comment:
Without these the library project fails to build on its own. This is needed for unit tests.